reason-react-native/src/apis/Style.md

```diff
@@ -4,4 +4,1298 @@ title: Style
 officialDoc: https://facebook.github.io/react-native/docs/style
 ---
 
-@todo
+Styles accepted by most components (thanks to a `style` prop) are defined using
+style functions that prepare required object for React Native. The style names
+and values usually match how CSS works on the web, except names are written
+using camel casing, e.g `backgroundColor` rather than `background-color`.
+
+The `style` prop accept what the style functions return. That's the simplest way
+to pass styles. You can also pass an array or list of styles - the last style in
+the array has precedence, so you can use this to mix & inherit styles.
+
+⚠️ _Note that when a component grows in complexity, it is often cleaner to use
+[`StyleSheet.create`](/bs-react-native/en/docs/apis/StyleSheet/) to define
+several styles in one place_.
+
+We have made different styles function because React Native have various
```

Comment:
Perhaps better to refer to the functions as constructors, (also fixing the plural)?
We have defined different style constructors because React Native have various

Reply:
Fixed